packages/wasm/package.json

```diff
@@ -20,7 +20,8 @@
     "lint:strict": "tsc --noEmit && eslint src --max-warnings 0",
     "postcompile": "touch ./wasm/.npmignore",
     "test": "vitest run",
-    "test:rust": "cd crate && wasm-pack test --headless --firefox -- --target wasm32-unknown-unknown --release --features 'mock-database'"
+    "test:rust": "cd crate && wasm-pack test --headless --firefox -- --target wasm32-unknown-unknown --release --features 'mock-database'",
+    "test:rust-unit": "cd crate && cargo test --release"
```

Comment:
issue: think rust vs rust-unit isn't the clearest differentiation. What if we did `test:wasm` vs `test:rust` (or `test:cargo`)?

Reply:
thanks, good idea - went with test:wasm and test:cargo here.